test/cqlpy/conftest.py

```diff
@@ -42,6 +44,15 @@ def pytest_addoption(parser):
     # presence.
     parser.addoption('--omit-scylla-output', action='store_true',
         help='Omit scylla\'s output from the test output')
+    parser.addoption('--no-minio', action="store", default=False, type=bool,
+                     help="Signal to not run S3 related tests")
```

Comment:
`--no-minio=true` is redundant, I would make this option a typeless one (flag), so user can just add `--no-mino` to the command line.